pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computebackendservice/globalcomputebackendserviceiap/create.yaml

```diff
@@ -28,7 +28,7 @@ spec:
   location: global
   iap:
     oauth2ClientIdRef:
-      external: iapidentityawareproxyclient-${uniqueId}
+      name: iapidentityawareproxyclient-${uniqueId}
```

Comment:
Nice; we'll now wait for readiness 👍

Reply:
@gemmahou I saw you resolved this - did I miss any follow-up offline?

Reply:
Ah I might resolve this by mistake. Yes with `external` the test will fail for a different reason. After I fixed this I got the "API disabled" error documented in the doc. I can reproduce it and provide the error message. It's something related to "invalid value for field".

Reply:
Thank you for sharing the context! We can keep track of this test failure offline.